base/src/main/java/io/spine/type/KnownTypes.java

```diff
@@ -253,13 +253,17 @@ public boolean apply(@Nullable TypeUrl input) {
      * @throws IllegalArgumentException if the name does not correspond to any known type
      */
     static GenericDescriptor getDescriptor(String typeName) {
+        checkNotNull(typeName);
+        checkArgument(!typeName.isEmpty(), "Type name cannot be empty");
```

Comment:
Seems that `Validate.checkNotEmptyOrBlank` may fit better than both of `check...` statements.